aws/data_source_aws_acm_certificate.go

```diff
@@ -169,6 +171,16 @@ func dataSourceAwsAcmCertificateRead(d *schema.ResourceData, meta interface{}) e
 	d.SetId(time.Now().UTC().String())
 	d.Set("arn", matchedCertificate.CertificateArn)
 
+	tags, err := keyvaluetags.AcmListTags(conn, d.Id())
```

Comment:
You'll need to pass `matchedCertificate.CertificateArn` here for the ID, instead of `d.Id()`. I'm asking internally why, but `d.Id()` is set to the UTC time a few lines above.

Reply:
done + tested to some degree:

```
--- PASS: TestAccAWSAcmCertificateDataSource_KeyTypes (139.64s)
```